### PR TITLE
Botsbase - Website

### PR DESCRIPTION
**Pull request by Roxza**
Hello, my english may be bad, I wanted to add my long-running website here and I guess it's happening Since I will add it for the first time, I do not have much information, I will be glad if you help.

Website: https://botsbase.net

### CODE REVIEW

File: websites/B/BotsBase/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "name": "Roxza",
+    "id": "716706383780904961"
+  },
+  "service": "botsbase.net",
+  "description": {
+    "tr": "Discord botlarını keşfedebileceğiniz ve botlarınızı paylaşabileceğiniz bir evren!",
+    "en": "A universe where you can discover Discord bots and share your bots!"
+  },
+  "url": "botsbase.net",
+  "version": "1.0.4",
```
Comment (Bas950) [inline websites/B/BotsBase/dist/metadata.json:12]: v1.0.0 on release

File: websites/B/BotsBase/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "author": {
+    "name": "Roxza",
+    "id": "716706383780904961"
+  },
+  "service": "botsbase.net",
```
Comment (Bas950) [inline websites/B/BotsBase/dist/metadata.json:6]: service name and foldername must be the same, and this is the name displayed on our website.

File: websites/B/BotsBase/presence.ts [hunk 3]
```diff
@@ -0,0 +1,88 @@
+const presence = new Presence({
+  clientId: "730486476059705354"
+});
+
+const browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (Bas950) [inline websites/B/BotsBase/presence.ts:5]: merge consts

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload proof of the presence working

**Comment by Roxza** [comment]
Hello again, I made the edits you want, the screenshots are below.

Home : https://i.imgur.com/xL1Fz7a.png
Add Bot : https://i.imgur.com/aD7CqNL.png
View Bot : https://i.imgur.com/fRXDuGQ.png
Bots Page : https://i.imgur.com/taJlrie.png
Search Page : https://i.imgur.com/SxBWhgr.png
Profile : https://i.imgur.com/7zITfyQ.png
Tags : https://i.imgur.com/S40ztyg.png
Vote : https://i.imgur.com/kaokEfi.png

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix codacy errors

**Comment by Bas950** [comment]
Also service name and folder name are still not the same

**Comment by Roxza** [comment]
> Fix codacy errors

If you tell me a little detail where the error is located, I will fix it faster.

**Comment by Bas950** [comment]
> > Fix codacy errors
> 
> If you tell me a little detail where the error is located, I will fix it faster.

At the bottom of your screen it says checks failed and then click on details at codacy

**Comment by Roxza** [comment]
all bugs solved

**Review by Bas950 — APPROVED** [review]

File: websites/B/BotsBase/presence.ts [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+});
+browsingStamp = Math.floor(Date.now() / 1000);
+let priceEls;
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "botsbase"
+  };
+	
+	presenceData.startTimestamp = browsingStamp;
```
Comment (ririxidev) [inline websites/B/BotsBase/presence.ts:12]: you could move startTimestamp to this object :/ but fine

**Review by ririxidev — APPROVED** [review]
Merry Christmas 🎉 and GJ

**Pull request merged**